packages/core/src/evaluation/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface EvaluationResult {
+  query: string;
+  contexts?: string[];
+  response: string;
+  score: number;
+  scoreSecondary?: number;
+  scoreSecondaryType?: string;
+  meta?: any;
+  passing: boolean;
+  feedback: string;
+}
+
+export interface BaseEvaluator {
+  evaluate(
+    query: string,
+    response: string,
+    contexts?: string[],
+    reference?: string,
+  ): Promise<EvaluationResult>;
+  evaluateResponse(
+    query: string,
+    response: Response,
+  ): Promise<EvaluationResult>;
```

Comment:
looks to me like this is not needed (if `evaluate` would use a parameter object)

Reply:
@marcusschiesser was missing a few lines of code that make the DX better with evaluateResponse

Reply:
wondering about change BaseEvaluator to a class then we can have this evaluateResponse on the base class